dicts/resolvers.txt

```diff
@@ -0,0 +1,986 @@
+103.20.188.35
```

Comment:
Where is this list from? It should be credited somewhere, and would also be good to know how we can keep it updated over time.

Reply:
I have added the following comment at the top of the file:
```
# Public DNS servers
# Reliability: 95% or higher
# Valid as of 2021-06-28
# Retrieved from https://public-dns.info/nameservers.csv
```

Also, the module will now automatically retrieve the latest dataset (only servers with 95% or higher reliability), and fall back to the local dictionary if the request fails or if there aren't any results.